ccf_transaction_processor/README.md

```diff
@@ -35,64 +36,32 @@ IMPORTANT: When installing CCF and PDO on the same VM for local testing,
 please install PDO first and then CCF. See [PDO docs](../docs) for
 detailed instructions on installing PDO.
 
-## Install CCF Base
+## Install CCF
```

Comment:
Overall, I see a bunch of interesting instructions, but ... none of these is in docker files.
I am assuming the CI/CD switched completely to using the docker container provided for building CCF apps.
If that's so, why don't we just get rid of this and provide instructions to use the docker container instead?
What's the advantage in downloading and installing CCF locally?

Reply:
docker only works with TP in simulation mode. does not support TP in HW mode.

Reply:
at least in part because i don't want to run it in docker :)